idioms/coercion-arguments.md

```diff
@@ -82,7 +85,7 @@ Running this example using our function declared with an argument type `&str` wi
 curious has three consecutive vowels!
 ```
 
-However, this example will not run when our function is declared with an argument type `&String`.  This is because string slices are a `&str` and not a `&String`.  One way you can think about why this is the case: to have a `&String` you will need a reference to a `String` object which requires three things: a pointer to your data, the length of your data, and how much capacity you have left to insert new data.  This last entry doesn't make much sense when you are talking about string slices. A string slices only cares about a window of your data, needing only: a pointer to your data, and the length of your data slice.
+However, this example will not run when our function is declared with an argument type `&String`.  This is because string slices are a `&str` and not a `&String`.
```

Comment:
Should mention how `&str` can't be coerced to `&String`, because the other direction does work.

e.g. "This is because string slices are of type `&str`, which cannot be converted to `&String`"